test/stubs/context.js

```diff
@@ -24,6 +24,7 @@ class ContextStub {
     return {
       owner: 'necojackarc',
       repo: 'auto-request-review',
+      org: 'necojackarc',
```

Comment:
I am not sure this is right. I couldn't find docs on the context object or where one might find org from @necojackarc

Reply:
@necojackarc is my personal account, so it may not have a corresponding org but this value seems good enough for a stub 👍